src/lib/identity/api.ts

```diff
@@ -57,8 +75,7 @@ const getUserFromCookie = (): IdentityUserFromCache => {
 		if (userData && cookieData) {
 			const displayName = decodeURIComponent(userData[2]);
 			userFromCookieCache = {
-				id: parseInt(userData[0], 10),
-				primaryEmailAddress: userData[1], // not sure where this is stored now - not in the cookie any more
+				id: parseInt(userData[0], 10), // not sure where this is stored now - not in the cookie any more
```

Comment:
I think this comment applied to the `primaryEmailAddress` field!

```suggestion
				id: parseInt(userData[0], 10),
```

Reply:
Good spot!